### ISSUE TITLE
Macbook pro Mid 2012 

### ISSUE THREAD

**Issue by nandy2783** [BenSova/Patched-Sur#440]
the sd card reader is not working, probably because no ktext file is their to support it, request you to make the scared working as it worked in previous installations like catalina.

**Comment by cenon415** [comment]
I just tested this on my MacBook Pro (15-inch, Mid 2012) [macOS 11.2.3 (20D91)] and it is working fine for me. My Macbook Pro is detecting the SD Card as usual.

**Comment by ozziepadilla** [comment]
> the sd card reader is not working, probably because no ktext file is their to support it, request you to make the scared working as it worked in previous installations like catalina.

Run the patchet kexts again from the patchet Sur app , then reboot your Mac

**Comment by nandy2783** [comment]
> > the sd card reader is not working, probably because no ktext file is their to support it, request you to make the scared working as it worked in previous installations like catalina.
> 
> Run the patchet kexts again from the patchet Sur app , then reboot your Mac

i did as you said, ran the ktext patch again, rebooted , no luck, nothing even under Disk Utility (Show all Devices selected) but under "About This mac",
 "system information " i see this under card reader !

![Screenshot 2021-04-11 at 6 46 24 PM](https://user-images.githubusercontent.com/4641500/114305556-3a8ec700-9af6-11eb-81f7-6083058e76dd.png)

**Comment by ozziepadilla** [comment]
> > > the sd card reader is not working, probably because no ktext file is their to support it, request you to make the scared working as it worked in previous installations like catalina.
> > 
> > 
> > Run the patchet kexts again from the patchet Sur app , then reboot your Mac
> 
> i did as you said, ran the ktext patch again, rebooted , no luck, nothing even under Disk Utility (Show all Devices selected) but under "About This mac",
> "system information " i see this under card reader !
> 
> ![Screenshot 2021-04-11 at 6 46 24 PM](https://user-images.githubusercontent.com/4641500/114305556-3a8ec700-9af6-11eb-81f7-6083058e76dd.png)

Do you have a SDcard plugged ? If no then plug it in reboot the Mac and check it again in system information after rebooted but let the Mac to optimize before go to system information, did your wifi works ?

**Comment by nandy2783** [comment]
> > > > the sd card reader is not working, probably because no ktext file is their to support it, request you to make the scared working as it worked in previous installations like catalina.
> > > 
> > > 
> > > Run the patchet kexts again from the patchet Sur app , then reboot your Mac
> > 
> > 
> > i did as you said, ran the ktext patch again, rebooted , no luck, nothing even under Disk Utility (Show all Devices selected) but under "About This mac",
> > "system information " i see this under card reader !
> > ![Screenshot 2021-04-11 at 6 46 24 PM](https://user-images.githubusercontent.com/4641500/114305556-3a8ec700-9af6-11eb-81f7-6083058e76dd.png)
> 
> Do you have a SDcard plugged ? If no then plug it in reboot the Mac and check it again in system information after rebooted but let the Mac to optimize before go to system information, did your wifi works ?

yes wifi, usb, bluetooth rest everything is working fine except no scared detection
Rebooted with scared plugged in.. still no success
also reapplied Ktext with sdcard plugged  and rebooted again still no luck!
also opened the logic board and cleaned the sdcard holder and cleaned the pins with alchol - still no luck( this i had done 2-3 days ago)

**Comment by 15wat** [comment]
I saw a similar post about 3 months ago so immediately checked my mid 2012 mbpr and sure enough sd cards were not recognized when inserted in the card slot. In clean my mac x there is a function "repair disk permissions" ran it and had no problems since. I'm posting this not because I am saying you should buy clean my mac or promoting it as there are many differences of opinion whether any of these types of apps are useful at all... However, it may point someone smarter than me in the right direction to resolve it. Having had previous exchanges with Ben re wifi kexts on my 2012 mbpr I was told the only (mid 2012 mbpr) kexts that are patched in post install are the wifi kexts and once I upgraded my the airport card to an "ac" version (which I did) no post install kexts would be installed.. This kind of negates the idea that you have a kext issue I think.
